pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -194,7 +194,7 @@ class Build {
         List testList = []
         def jdkBranch = getJDKBranch()
         def jdkRepo = getJDKRepo()
-        def openj9Branch = buildConfig.SCM_REF ? buildConfig.SCM_REF : "master"
+        def openj9Branch = buildConfig.SCM_REF && buildConfig.VARIANT == "openj9" ? buildConfig.SCM_REF : "master"
```

Comment:
Can we bracket this for clarity (hopefully this syntax is valid in groovy!)
```suggestion
        def openj9Branch = (buildConfig.SCM_REF && buildConfig.VARIANT == "openj9") ? buildConfig.SCM_REF : "master"
```

Reply:
Yes. It is more clear that way. please check again. Thanks